src/asset-picker/AssetPickerDropDown.js

```diff
@@ -35,26 +35,27 @@ export default class AssetPickerDropDown extends Component {
             router.push('asset-picker');
         }
     }
+    onClose(e) {
+        const { actions } = this.props;
+        e.stopPropagation();
```

Comment:
In that case, this can be added to the overlay in DropDown component, but not here.